cloudevents/working-drafts/xml-format.md

```diff
@@ -173,6 +182,13 @@ Example _(XML preamble and namespace definitions omitted for brevity)_:
 </event>
```

Comment:
This comment is probably more appropriate for the main spec, and I'll open a PR if people think we need it, but in formats like XML where the same element name can appear more than once do we need to add something along the lines of "Each CE attribute MUST only appear at most once"? I'm not sure we ever actually come right out and say each is a singleton - though it certainly is implied.

This popped in my head because I was trying to think of a case where someone may want to extend a CE attribute with an XML attribute and duplicating the CE attribute came to mind, e.g.:
```
<id foo:from="src1">123</id>
<id foo:from="src2">321</id>
```
Thoughts?

Reply:
Definitely! (I think I've got that in my test suite... I should really make sure that everything in there is covered at some point.) Coming up in another commit...

Reply:
Done. What about multiple elements in the same batch having the same ID+source? That might be overkill.

Reply:
I'm leaning towards not touching the ID+source issue. The main spec already says "Producers MUST ensure that source + id is unique for each distinct event." so I think that should cover it. Since a replayed event could happen across two different messages, that spec text should be enough to cover those cases. I don't view the same event in a single batch as much different - just a less verbose case.

Reply:
+1 on this .. nice catch ...

Reply:
Note: in my follow-up PR I haven't included this part, as @duglin's change to the main spec covers it.